features/providers/means_report.feature

```diff
@@ -168,6 +169,174 @@ Feature: Means report
       | question |
       | Uploaded bank statements |
 
+  Scenario: For a non-passported enhanced bank statement upload journey
+    Given the feature flag for enhanced_bank_upload is enabled
+    And I have completed a non-passported employed application with enhanced bank statement uploads
+    When I view the means report
+
+    Then the following sections should exist:
+      | tag | section |
+      | h2  | Client details |
+      | h2  | Proceeding eligibility |
+      | h2  | Passported means |
+      | h2  | Income result |
+      | h2  | Income |
+      | h3  | Employment income |
+      | h2  | Outgoings |
+      | h2  | Deductions |
+      | h2  | Caseworker Review |
+      | h2  | Capital result |
+      | h2  | Property, savings and other assets |
+      | h3  | Property |
+      | h3  | Vehicles |
+      # | h2  | Which bank accounts does your client have? | * TODO: awaiting flow change for journey (AP-3536)
```

Comment:
+1 (I think we should be creating tasks in JIRA rather than writing TODOs.)

Reply:
as above